thirdeye/thirdeye-frontend/app/pods/manage/alerts/performance/route.js

```diff
@@ -3,18 +3,30 @@
  * @module self-serve/create/route
  * @exports alert create model
  */
-import { getWithDefault } from '@ember/object';
 
-import { isPresent } from '@ember/utils';
-import Route from '@ember/routing/route';
+import RSVP from 'rsvp';
```

Comment:
don't think this is necessary since we're importing hash and allSettled in https://github.com/linkedin/pinot/pull/2613/files#diff-079b46af257352395a99cf63940b5e30R14